docs/api/swagger.json

```diff
@@ -69,47 +75,927 @@
+        "security": [],
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/groupId"
+          }
+        ],
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      },
+      "put": {
+        "summary": "Modify group",
+        "tags": [
+          "group"
+        ],
+        "operationId": "modifyGroup",
+        "security": [],
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/groupId"
+          }
+        ],
+        "requestBody": {
+          "$ref": "#/components/requestBodies/ModifyGroup"
+        },
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      }
+    },
+    "/languagePackage/{languagePackageId}": {
+      "delete": {
+        "summary": "Delete language package",
+        "tags": [
+          "language package"
+        ],
+        "operationId": "deleteLanguagePackage",
+        "security": [],
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/languagePackageId"
+          }
+        ],
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      },
+      "put": {
+        "summary": "Modify a language package",
+        "tags": [
+          "language package"
+        ],
+        "operationId": "modifyLanguagePackage",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/languagePackageId"
+          }
+        ],
+        "requestBody": {
+          "$ref": "#/components/requestBodies/ModifyLanguagePackage"
+        },
+        "security": [],
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      }
+    },
+    "/info": {
+      "get": {
+        "summary": "Get server info",
+        "tags": [
+          "info"
```

Comment:
maybe you can put this into the misc tag. *(Dont forget to move this up to hold it cleanly)*